app/models/comment.rb

```diff
@@ -0,0 +1,2 @@
+class Comment < ApplicationRecord
+end
```

Comment:
- According to the ERD diagram that was provided, `comments` belong to `users` and `posts`. Kindly set up a `belongs_to` association to handle this. 😉 

- Let's add a method that updates the comments counter for a post. It would be nice to have this handled in the model and have the controllers and views as thin as possible. Here 👇 is an example of how this can be done.

```
def <method_name>
  post.increment!(:comments_counter)
end
```